src/ripple/basics/tagged_integer.h

```diff
@@ -157,6 +163,19 @@ class tagged_integer
         return tagged_integer (lhs.m_value - rhs);
     }
 
+    template <class OtherInt>
+    friend
+    typename std::enable_if <
+        std::is_integral <OtherInt>::value &&
+            sizeof (OtherInt) <= sizeof (Int),
```

Comment:
Just bringing in the perspective from `<chrono>` on this:

It seems unusual to have `%` but not `/`.

If we have `/`, and if `tag/int` is supported, then `tag%int` should be supported, else it should not.

`<chrono>` support division and modulo of durations with ints:

  * 12 seconds / 4 == 3 seconds
  * 12 seconds / 4 seconds == 3

From these, the units of '%' are derived algebraically.